frontend/src/components/Dialog/Dialog.vue

```diff
@@ -35,29 +35,14 @@
     </vue-final-modal>
 </template>
 
-<script>
-export default {
-    inheritAttrs: false,
-    emits: ['confirm'],
-    props: {
-        message: {
-            type: String,
-            default: '',
-        },
-        title: {
-            type: String,
-            default: '',
-        },
-        confirmText: {
-            type: String,
-            default: 'Confirm',
-        },
-        cancelText: {
-            type: String,
-            default: 'Cancel',
-        },
-    },
-};
+<script setup lang="ts">
```

Comment:
We want to add `inherit-attrs="false"` here, <script setup inherit-attrs="false" lang="ts">

Reply:
Thanks! I have changed it.